### DISCUSSION TITLE
checkver uses wrong url

### DISCUSSION THREAD

**Discussion by ImMakSpy** [ScoopInstaller/Scoop#5654] [Help and Questions]
Hello
I'm trying to implement a manifest for my canon ts8300 driver, where checkver is
     "checkver": {
        "url": "https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?type=drivers&detailId=tcm:84-1876397&os=Windows%2010%20%2864-bit%29&language=&productTcmUri=tcm:84-1876207/",
        "regex": "Dateiversion.*(\\d+\\.\\d+)"
    },
There I've the problem that, when I execute the version check (checkver.ps1) I got the following error:
- couldn't match 'Dateiversion.*(\d+\.\d+)' in https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?type=drivers&detailId=tcm:84-1876397&os=Windows%2010%20%2864-bit%29&language=&productTcmUri=tcm:84-1876207/

At first I thought the regex is wrong. But after I've no clue what's wrong in the regex, I found out, that the content is from
- https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html

and not from
- https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?type=drivers&

detailId=tcm:84-1876397&os=Windows%2010%20%2864-bit%29&language=&productTcmUri=tcm:84-1876207/".

I have the same problem, when I call
- $WebResponse = Invoke-WebRequest "https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?type=drivers&detailId=tcm:84-1876391&os=Windows%2010%20(64-bit)&productTcmUri=tcm:84-1876207"
- $WebResponse = Invoke-RestMethod -Uri 'https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?type=drivers&detailId=tcm:84-1876391&os=Windows%2010%20(64-bit)&productTcmUri=tcm:84-1876207' -Method GET
- curl -L "https://www.canon.at/support/consumer/products/printers/pixma/ts-series/pixma-ts8350.html?detailId=tcm%3A84-1876391&os=Windows+10+&type=drivers"

Has anyone an idea, how my problem can be solved?
I hope someone can help me.
I'm thankful for every reply.

**Comment by r15ch13** [comment]
The content is loaded via JavaScript, so you need to open the developer tools (press F12) and then you get what you need from the network tab.

https://www.canon.at/supportproduct/tabcontent/?type=drivers&os=Windows%2010%20(64-bit)&productTcmUri=tcm:84-1876207&pageTcmUri=2269755

And here a working regex: https://regex101.com/r/34zu1y/1
`Dateiversion:[^\d]+([\d.]+)`
Match everything that is not a number `[^\d]+` and then match the version including the dot `([\d.]+)`
